libs/shared/src/lib/constants/content-constants.ts

```diff
@@ -0,0 +1,15 @@
+import { Validators } from '@angular/forms';
+
+export class ContentConstants {
+    public static VAL_MIN_TEXT_LENGTH = Validators.minLength(3);
+    public static VAL_MAX_TITLE_LENGTH = Validators.maxLength(100);
+    public static VAL_MAX_DESC_LENGTH = Validators.maxLength(250);
+
+    public static MIN_GENRES = 1;
+    public static VAL_MIN_GENRES = Validators.minLength(ContentConstants.MIN_GENRES);
+    public static MAX_GENRES = 3;
+    public static VAL_MAX_GENRES = Validators.maxLength(ContentConstants.MAX_GENRES);
+
+    public static MAX_TAGS = 5;
+    public static VAL_MAX_TAGS = Validators.maxLength(ContentConstants.MAX_TAGS);
+}
```

Comment:
@Figments, correct me if I'm misremembering, but I think `libs/shared` is meant to be code we can share between back- and frontend, i.e. it doesn't take dependencies on Angular.

Everything that follows is just a nitpick:
This works fine, but it would be slightly more idiomatic for them to be just naked `export const`, i.e.
```TypeScript
export const VAL_MIN_TEXT_LENGTH: number = 3;
// etc
```

...if you wanted them to still live inside a `ContentConstants` object to make their usage a little more obvious, you could also do something like
```TypeScript
export const ContentConstants: {
  valMinTextLength: 3,
  valMaxTitleLength: 100,
  // etc...
}
```
...though the actual values themselves would still be mutable. You could probably do something with namespaces too, but I think we're trying to avoid those throughout the codebase, for consistency reasons.

Reply:
I was iffy on making Validators constants anyway. I'll just have the numbers as constants.